packages/fxa-content-server/app/scripts/views/mixins/token-code-experiment-mixin.js

```diff
@@ -1,67 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was no longer used and should have been removed with another PR that was merged earlier in the week.